### PR TITLE
README.md and German translations

### PR DESCRIPTION
**Pull request by lucode**
I did add a README.md file for each module and the German translation files.
The root README.md file contains the installation tutorial now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,40 @@
...
+The project is managed by a growing worldwide community of volunteers that are willing to contribute to the development and documentation to make this vision real.
 
-Odoo Modules allowing to transform it into a powerful tool for managing communities
+## About ##
+Communities Verticalizations provides solutions to manages a wide range of needs of collaborative project and organisations. It bundles helpful existing odoo apps as Blog, Forum together with new modules to finally get a versatile and interconnected selection of community related apps.
+The new apps are covering almost any basic needs to manage many types of ressurces in Non Profit Organisations. So it includes apps für Crowdfunding, Coworking, Complementary Currencies and a powerful Marketplace. The currency (eWallet) modules are fully integrated into the financial part of odoo so it allows to perform many financial and monetary actions. You can use it for issuing currencies, paying resources from the marketplace or a shop, sending/receiving digital cash aso.. Because the eWallet is able to handle multi-currencies, the framework is open to include other transaction engines as Cyclos or Cryptocurrency framework one day.
```
Comment (YannickB) [inline README.md]: für -> for
not sure it's a good idea to speak about cyclos, this is a trademark and I'm not sure it's really a transaction engine, more a marketplace like us.

**Comment by YannickB** [comment]
Thank you, I just added a little comment to resolve on the installation documentation.

Regarding the technical README.md on each module, I'm not really sure it's really useful but I'll accept it anyway.

File: README.md (hunk 1 above)
Comment (lucode) [inline README.md]: correction is done.
Cyclos is both, a full framework with marketplace aso. and a transaction 
engine too.
There are many sites who have a Python / Django frontend with Cyclos as 
Backend.
http://www.qoin.org/how-we-work/solutions/qoinware/
I don't see any problem to cite the name Cyclos. They would be happy if 
they get a odoo frontend too.

Lucas

Am 24.11.2014 um 10:33 schrieb Yannick Buron:

Comment (YannickB) [inline README.md]: Ok for me then

**Comment by YannickB** [comment]
Ok for me, I merge the pull request

**Pull request merged**